docker/protect.json

```diff
@@ -1,36 +1,147 @@
 {
```

Comment:
I thought we agreed you're going to add signed links from redwood for each input reference file. Hardcoding the values in the wrapper is a very bad way to do this.